judgments/templatetags/search_results_filters.py

```diff
@@ -0,0 +1,37 @@
+import re
+
+from django import template
+
+from judgments.utils import preprocess_query
+
+register = template.Library()
+
+not_alphanumeric = re.compile("[^a-zA-Z0-9]")
+
+
+def preprocess_title(string):
+    return preprocess_query(string).lower()
+
+
+def preprocess_ncn(string):
+    return re.sub(not_alphanumeric, "", preprocess_query(string).lower())
```

Comment:
Wondering if we want to normalise spaces rather than removing them -- we might highlight matches which split over word boundaries (Manbat v Man == Man v Batman).

Possibly not -- the search filter should remove those, and "BBC" and "B. B. C." matching is good.